app/components/pokemon.jsx

```diff
@@ -5,7 +5,7 @@ import { faInfo }                   from '@fortawesome/free-solid-svg-icons';
 import { useDispatch, useSelector } from 'react-redux';
 
 import { ReactGA }                        from '../utils/analytics';
-import { createCaptures, deleteCaptures } from '../actions/capture';
+import { createCaptures, deleteCaptures, markPending } from '../actions/capture';
```

Comment:
it doesn't look like `markPending` is being used in this file, so let's remove it as an import